content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -723,13 +723,8 @@ Only statically provisioned volumes are supported for alpha release. Administrat
 
 ## Volume Snapshot and Restore Volume from Snapshot Support
 
-{{< feature-state for_k8s_version="v1.17" state="beta" >}}
-
 Volume snapshot feature was added to support CSI Volume Plugins only. For details, see [volume snapshots](/docs/concepts/storage/volume-snapshots/).
```

Comment:
I'd like to see this sentence revised to suit the now-stable feature.

Reply:
How about `Volume snapshot feature is graduated in the 1.20 release.  It only supports out-of-tree CSI Volume Plugins as in-tree Volume Plugins are deprecated.  See the doc [here](https://github.com/kubernetes/community/blob/master/sig-storage/volume-plugin-faq.md) for details.`?